src/components/graph-view.js

```diff
@@ -1295,6 +1311,57 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
     ReactDOM.render(anyElement, nodeContainer);
   }
 
+  asyncElevateNodeAndEdges(node: INode) {
+    const asyncElevateNode = node =>
+      requestAnimationFrame(() => this.syncElevateNode(node));
+    const asyncElevateEdge = edge =>
+      requestAnimationFrame(() => this.syncElevateEdge(edge));
+
+    const nodeMapNode = this.getNodeById(node[this.props.nodeKey]);
+
+    asyncElevateNode(node);
+
+    nodeMapNode.incomingEdges.forEach(asyncElevateEdge);
+    nodeMapNode.outgoingEdges.forEach(asyncElevateEdge);
+  }
+
+  syncElevateEdge(edge: IEdge) {
+    if (!this.entities) {
+      return;
+    }
+
+    // We have to use the 'custom' id when we're drawing a new node
+    const idVar = edge.target ? `${edge.source}-${edge.target}` : 'custom';
+    const id = `edge-${idVar}`;
+    const containerId = `${id}-container`;
+    const edgeContainer:
+      | HTMLElement
+      | Element
+      | null = this.entities.querySelector(`#${containerId}`);
+
+    if (edgeContainer) {
+      this.entities.appendChild(edgeContainer);
+    }
+  }
+
+  syncElevateNode(node: INode) {
```

Comment:
we already elevate node on move in `components/node.js` should we centralize all the elevation here, potentially removing the need to re-trigger the `click` event?

Reply:
I moved the code out of `components/node.js` into here -- it's no longer in `components/node.js`.

Reply:
ahh nice